docs/_docs/installation.md

```diff
@@ -111,6 +113,34 @@ Check out the [troubleshooting](../troubleshooting/) page or
 
 </div>
 
+## Install on Ubuntu Linux {#ubuntu}
+
+Before we install Jekyll, we need to make sure we have all the required
+dependencies.
+
+```sh
+sudo apt-get install ruby ruby-dev build-essential
+```
+
+It is best to avoid installing Ruby Gems as the root user. Therefore, we need
+to set up a gem installation directory for your user account. Run the following
+commands to add some configuration to your `.bashrc` file.
+
+```sh
```

Comment:
What if users have already a custom PATH defined in `.bashrc`?

What about using heredoc to make this a single command?

```sh
cat > .bashrc <<- GEMS
# Install Ruby Gems to ~/gems
export GEM_HOME=$HOME/gems
export PATH=$HOME/gems/bin:$PATH
GEMS
```

Reply:
It's fine if a user already has a custom path - the `PATH` modification appends the existing `$PATH` at the end.

I'm torn on the heredoc, but certainly happy to include it if you think it's better (I do think it's cleaner). 2 things to consider though:

* Heredoc will be more confusing if a user copy/pastes commands line-by-line (since they won't get a prompt back after pasting the first line).
* It will be harder for users who are unfamiliar with bash to look up the heredoc command and see what it's doing.

Reply:
Don’t make novice users lookup heredoc